### ISSUE TITLE
Bucket listing output not in json and looks like log output

### ISSUE THREAD

**Issue by ramya-c3** [noobaa/noobaa-core#7594]
### Environment info
- NooBaa Version: 5.15
- Platform: Standalone Noobaa

### Actual behavior
1.# node src/cmd/manage_nsfs bucket list 

Oct-31 2:46:46.503 [/2193050]   [LOG] CONSOLE::  [ { name: 'b1' } ]

### Expected behavior
1. it should be proper json

**Comment by liranmauda** [comment]
@ramya-c3 
#7568 changed that, now when we are listing, we will print a list in the Array, and not a JSON output (it still looks like a log.
the output will be :
```
$ node src/cmd/manage_nsfs bucket list --config_root /Users/liranmauda/Documents/nsfs_config_dir
Nov-14 14:40:02.270 [/65520]   [LOG] CONSOLE:: Bucket list [ 'bucket', 'bucket1' ]
```

I do not think that a list should be a JSON output. 

Please use the latest version.
Let me know if you have something to add or if we can close this one.

**Comment by romayalon** [comment]
@ramya-c3 should be fixed with noobaa-core-5.15.0-20231207-master.el8.x86_64.rpm, please validate and close if fixed.
